### DISCUSSION TITLE
Translations for „open“ need context

### DISCUSSION THREAD

**Discussion by carlosmintfan** [community/community#108315] [Mobile] [answered]
### Select Topic Area

Bug

### Body

The english word _open_ is both a verb and an adjective, which leads to wrong german translations like „öffnen“ (verb) instead of „offen“ (adjective) and/or vice versa. Please provide feedback for the translators to make them know if they should provide a translation for the verb or adjective. Thanks!

**Comment by HenSquared** [comment, marked as answer]
Hi,

Thanks for the feedback! I've opened an issue with our team to go through and identify incorrect uses of öffnen and offen.

If you have specific examples of the incorrect usage in the app, can you please share them here? That would be very helpful. :) 

Thanks!

**Reply by carlosmintfan** [reply]
Hi @HenSquared,
thanks for your message. Unfortunately I can't find the issue anymore although I encountered in the past. It seems to have been fixed in a recent update. If I reencouter this in the future, I'll report it. Sorry for the noise :see_no_evil:

**Reply by HenSquared** [reply]
No worries @carlosmintfan , this still helps us keep a lookout for those issues. :)
